package.json

```diff
@@ -40,11 +40,10 @@
     "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
     "@semantic-release/npm": "12.0.1",
+    "@stylistic/eslint-plugin-js": "2.1.0",
```

Comment:
contains the new `indent` rules (see https://eslint.org/docs/latest/rules/indent)